src/components/VideoPlayerContexts/VideoPopoverMenuContext.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React, {useCallback, useContext, useMemo, useState} from 'react';
+import _ from 'underscore';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useLocalize from '@hooks/useLocalize';
+import fileDownload from '@libs/fileDownload';
+import * as Url from '@libs/Url';
+import CONST from '@src/CONST';
+import {usePlaybackContext} from './PlaybackContext';
+
+const VideoPopoverMenuContext = React.createContext(null);
+
+function VideoPopoverMenuContextProvider({children}) {
+    const {currentVideoPlayerRef} = usePlaybackContext();
+    const {translate} = useLocalize();
+    const [currentPlaybackSpeed, setCurrentPlaybackSpeed] = useState(CONST.VIDEO_PLAYER.PLAYBACK_SPEEDS[2]);
+
+    const updatePlaybackSpeed = useCallback(
+        (speed) => {
+            setCurrentPlaybackSpeed(speed);
+            currentVideoPlayerRef.current.setStatusAsync({rate: speed});
```

Comment:
This line caused this bug:
https://github.com/Expensify/App/issues/40646

Here's the RCA:
https://github.com/Expensify/App/issues/40646#issuecomment-2067790768